Makefile.am

```diff
@@ -164,7 +164,12 @@ dist/%/Makefile.deps: $(WEBPACK_CONFIG)
 
 WEBPACK_MAKE = NODE_ENV=$(NODE_ENV) SRCDIR=$(srcdir) BUILDDIR=$(builddir) \
 	       $(srcdir)/tools/missing $(srcdir)/tools/webpack-make
+if MAINTAINER_MODE
 WEBPACK_RULE = $(V_WEBPACK) $(WEBPACK_MAKE)
+else
+WEBPACK_RULE = @echo "Skipping webpack update in non-maintainer mode: $(@:dist/%/Makefile.deps=%)"; \#
+endif
```

Comment:
Yes, as `make clean` removes dist/`. It shouldn't do that as we ship `dist/` in our tarballs. I  suppose we should fix that to only remove dist/ stuff on `distclean`, not `clean`?